src/main/webapp/app/exercises/quiz/manage/statistics/quiz-statistic.route.ts

```diff
@@ -53,4 +53,49 @@ export const quizStatisticRoute: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
+    {
+        path: ':courseId/exams/:examId/exercise-groups/:exerciseGroupId/quiz-exercises/:exerciseId/quiz-point-statistic',
```

Comment:
Can we define those in the exam routes file or is that not possible?

Reply:
you mean the exam-mangement,route, right? I can move them into there, it won't cause a problem. I just thought that statistics has separate file, maybe I can put the exam routes in there. Should I define them in exam-management?

Reply:
Hm, I thought it would make sense to have all exam routes together, so I'd prefer having those routes in the exam-management.route.

Reply:
hmm, alright!! 👍